src/backend/d3d12/FramebufferD3D12.cpp

```diff
@@ -87,8 +79,7 @@ namespace d3d12 {
                 device->GetD3D12Device()->CreateRenderTargetView(device->GetCurrentTexture().Get(), &rtvDesc, rtvHandle);
             }
         }
-
-        if (usingBackbuffer) {
+        if (subpassInfo.depthStencilAttachmentSet) {
             args.dsv = dsvHeap.GetCPUHandle(0);
```

Comment:
And then we'll need to remember the depthStencilIndex for each subpass so we can get the correct handle here.